spyder/widgets/ipythonconsole/shell.py

```diff
@@ -70,6 +71,9 @@ def __init__(self, ipyclient, additional_options, interpreter_versions,
         # To save kernel replies in silent execution
         self._kernel_reply = None
 
+        # To set the color of the matched parentheses
+        self.set_bracket_matcher_color_scheme(self.syntax_style)
```

Comment:
I'm not sure it is ok. I've tried to keep it as short as possible. What do you guys think?

```
# Set the color of the matched parentheses here since the qtconsole
# uses a hard-coded value that is not modified when the color scheme is
# set in the qtconsole constructor.
```